yubioath/gui/view/dialogs.py

```diff
@@ -0,0 +1,26 @@
+from PySide.QtGui import QCheckBox, QDialog, QGridLayout, QLabel, \
```

Comment:
Lost of other files in /view/ are dialogs, it's weird that this file is named "dialogs.py"